ztp/ztp-policy-generator/Makefile

```diff
@@ -0,0 +1,33 @@
+PWD := $(shell pwd)
+BASE_DIR := $(shell basename $(PWD))
+
+# Keep an existing GOPATH, make a private one if it is undefined
+GOPATH_DEFAULT := $(PWD)/.go
+export GOPATH ?= $(GOPATH_DEFAULT)
+GOBIN_DEFAULT := $(GOPATH)/bin
+export GOBIN ?= $(GOBIN_DEFAULT)
+export PATH := $(PATH):$(GOBIN)
+XDG_CONFIG_HOME=./
+KUSTOMIZE_PLUGIN_HOME ?= $(XDG_CONFIG_HOME)/kustomize/plugin
+
+.PHONY: build test test-kustomize all clean
+build:
+	go build -mod=vendor -o policygenerator/PolicyGenerator ./policygenerator/
```

Comment:
Additionally, we should rename the directory from `policygenerator` to `PolicyGenerator`, and then we could remove the `-o ...` and rely on golang's implicit executable naming.

In fact, only kustomize requires that this executable be named `PolicyGenerator`, so we could build it as something more go-normal, like `policyGenerator`, and just rename it when we do the copy for kustomize.